mlflow/deployments/mlflow/__init__.py

```diff
@@ -30,7 +30,24 @@
 @experimental
 class MLflowDeploymentClient(BaseDeploymentClient):
```

Comment:
@harupy Should this be `MlflowDeploymentClient` for consistency with `MlflowClient`?

Reply:
Filed https://github.com/mlflow/mlflow/pull/10500